components/automate-ui/src/app/modules/infra-proxy/edit-environment-attribute-modal/edit-environment-attribute-modal.component.ts

```diff
@@ -275,8 +266,7 @@ export class EditEnvironmentAttributeModalComponent implements OnChanges, OnInit
   private toDisplay(cookbookVersions: Array<CookbookConstraintGrid> = []) {
     const current = {};
     cookbookVersions.forEach((element) => {
-      current[element.name] =
-        `${element.operator}` + ' ' + `${element.version}`;
+      current[element.name] = `${element.operator} ${element.version}`;
```

Comment:
Author note: correcting an inappropriate use of backticks (which interpolate objects into string) -- they should not even have been used at all if string concatenation was being used with it.